src/content/en/fundamentals/native-hardware/capturing-images/index.md

```diff
@@ -2,116 +2,242 @@ project_path: /web/_project.yaml
+<img class="attempt-right" src="images/ios-chooser.png" />
+<img class="attempt-right" src="images/android-chooser.png" />
+
+The data can then be attached to a `<form>` or manipulated with JavaScript by
+listening for an `onchange` event on the input element and then reading
 the `files` property of the event `target`.
 
-### Capture a single frame
+    <input type="file" accept="image/*" id="camera">
+    <script>
+      const camera = document.getElementById('camera');
+
+      camera.addEventListener('change', (e) => doSomethingWithFiles(e.target.files));
+    </script>
+
+The `files` property is a `FileList` object, which we'll talk more about later.
+
+You can also optionally add the `capture` attribute to the element, which indicates to the browser
+that you prefer getting an image from the camera.
+
+    <input type="file" accept="image/*" capture>
+    <input type="file" accept="image/*" capture="user">
+    <input type="file" accept="image/*" capture="environment">
+
+Adding the `capture` attribute without a value let's the browser decide which camera to use, while
+the `"user"` and `"environment"` values tell the browser to prefer the front and rear cameras,
+repectively.
 
-Getting access to the image file is simple.
+Be aware, however, that on Android this means that
+the user will no longer have the option of choosing an existing picture. The system camera app will
+be started directly, instead.
 
-    <input type="file" accept="image/*" capture="camera" id="camera">
-    <img id="frame">
+### Drag and drop
+If you are already adding in the ability to upload a file, there are a couple of easy ways that you
+can make the user experience a little richer.
+
+The first is to add a drop target to your page that allows the user to drag in a file from the
+desktop or another application.
+
+    <div id="target">You can drag an image file here</div>
     <script>
-      var camera = document.getElementById('camera');
-      var frame = document.getElementById('frame');
+      const target = document.getElementById('target');
 
-      camera.addEventListener('change', function(e) {
-        var file = e.target.files[0]; 
-        // Do something with the image file.
-        frame.src = URL.createObjectURL(file);
+      target.addEventListener('drop', (e) => {
+        e.stopPropagation();
+        e.preventDefault();
+
+        doSomethingWithFiles(e.dataTransfer.files);
+      });
+
+      target.addEventListener('dragover', (e) => {
+        e.stopPropagation();
+        e.preventDefault();
+
+        e.dataTransfer.dropEffect = 'copy';
       });
     </script>
 
-Once you have access to the file you can do anything you want with it. For 
-example, you can:
+Similar to the file input, you can get a `FileList` object from the `dataTransfer.files` property of
+the `drop` event;
+
+The `dragover` event handler let's us signal to the user what will happen if they drop the file they
```

Comment:
Maybe show what looks like the `dropEffect` signal. 
And/or link to https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/dropEffect